mod/wikirate/set/type_plus_right/metric_value/source.rb

```diff
@@ -1,9 +1,10 @@
 format :html do
   view :editor do |args|
-    
     if card.new?
-      source = Card.new type_code: :source, name: "new source"
-      subformat(source)._render_content_formgroup(:hide=>'header help',:buttons=>"")
+      source = Card.new type_code: :source, name: 'new source'
+      subformat(source)._render_content_formgroup(hide: 'header help',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.